app/models/user.rb

```diff
@@ -58,6 +58,14 @@ def self.from_oauth_hash(auth_hash)
     user
   end
 
+  def client_model
+    Proposal.client_model_for(self)
+  end
+
+  def client_model_slug
+    client_model.to_s.underscore.tr("/", "_")
```

Comment:
given the current setup (just two clients) do you think it makes sense to set this via the client slug? I was not aware of future plans or potential plans to add more clients within each namespace.

```

def client_model_slug
  if client_slug == "ncr"
      "ncr_work_order"
  elsif client_slug == "gsa"
    "gsa18f_procurement"
  end
end
```